notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<p>We need to explain why those InputCUTInfos are used and why. Otherwise it is not possible to change and learn from this...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>added description</p>